modules/core/src-2/kinds/package.scala

```diff
@@ -0,0 +1,21 @@
+package smithy4s
+
+package object kinds {
+
+  // format: off
+  type FunctorAlgebra[Alg[_[_, _, _, _, _]], F[_]] = Alg[Kind1[F]#toKind5]
+  type FunctorInterpreter[Op[_, _, _, _, _], F[_]] = PolyFunction5[Op, Kind1[F]#toKind5]
+  type BiFunctorAlgebra[Alg[_[_, _, _, _, _]], F[_, _]] = Alg[Kind2[F]#toKind5]
+  type BiFunctorInterpreter[Op[_, _, _, _, _], F[_,_]] = PolyFunction5[Op, Kind2[F]#toKind5]
```

Comment:
Whats your take on that FunctorTransformer/FunctorTransformation would be a more fitting type alias for FunctorInterpreter 
@Baccata

Reply:
I like the idea of `FunctorProjection5` and `BiFunctorProjection5`.  Seems more fitting than Interpreter in this use case. Taking the free-monad in consideration the current naming also makes sense.

Reply:
I like the idea of FunctorProjection as well. I am usually fine with long names though if they are more accurate or descriptive.

Reply:
I like `FunctorInterpreter` because I feel like the "interpreter" part sort of explains what it _does_, not just what it _is_ technically - a more semantic name than "projection". I'm fine with either though